video/out/drm_common.c

```diff
@@ -1099,3 +1100,451 @@ void drm_pflip_cb(int fd, unsigned int msc, unsigned int sec,
     *closure->waiting_for_flip = false;
     talloc_free(closure);
 }
+
+const char* drm_format_string(uint drm_format) {
```

Comment:
code style: function brace on next line
code style: pointer `*` on right side

also the `uint` shorthand is used nowhere in the mpv codebase, this should be either `unsigned int` or another appropriate type (`uint32_t` for drm formats)